### DISCUSSION TITLE
Confused on install/Question about install

### DISCUSSION THREAD

**Discussion by RyuAzuku** [trickerer/Trinity-Bots#360] [General]
Currently using PrivateDonuts Ultimate Repack for 3.3.5a and I wanted to install NPCBots along side it, but the instructions aren't clear at all for what I'm supposed to use for the required commands. Does anyone have any help?

**Comment by trickerer** [comment]
You cannot install any patch if you use a repack since with patch you modify source files and build the core yourself, there is no other way.
If you want a repack with NPCBots look for one which includes them

**Reply by RyuAzuku** [reply]
Alright, thank you for the reply.
